src/types/timestamp.rs

```diff
@@ -250,6 +272,16 @@ impl Timestamp {
             .into_builder();
         FractionalSecondSetter { builder }
     }
+
+    /// Returns the offset that has been specified in the [Timestamp].
+    pub fn offset(&self) -> Option<FixedOffset> {
+        self.offset
```

Comment:
I'd like to avoid returning an external type as part of the public API. (`FixedOffset` lives in `chrono`.) I think this should be:
```
pub fn offset_minutes(&self) -> Option<i32>;
```
(Be sure to document what positive and negative minutes mean.)

Reply:
Sure. That makes sense. I will change this.